src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -16,8 +16,7 @@ namespace System.Diagnostics
 {
     public partial class Process : IDisposable
     {
-        private static readonly UTF8Encoding s_utf8NoBom =
-            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false);
+        private static readonly UTF8Encoding s_utf8NoBom = Encoding.Default;
```

Comment:
We don't need the field at all... the three call sites in this file can just use Encoding.Default directly.